src/Slider/Slider.js

```diff
@@ -0,0 +1,150 @@
+import { Box, Slider as MuiSlider, Typography } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+/**
+ * Discrete slider component
+ */
+export default function Slider({
+  displayCurrentValue = "auto",
+  max = 10,
+  min = 1,
+  onChange = () => {},
+  onChangeCommitted = () => {},
+  orientation = "horizontal",
+  predefValues = [],
+  labels = true,
+  labelPosition = "bottom",
+  step = 1,
+  title,
+  value
+}) {
+  // get ticks labels if they are not pre defined
+  const range = (start, stop, step) =>
+    Array.from(
+      { length: (stop - start) / step + 1 },
+      (_, i) => start + i * step
+    );
+  // show/hide labels
+  let marks = [];
+  let stepSize = step;
+  if (labels) {
+    if (!predefValues.length) {
+      range(min, max, stepSize).map(item => {
+        const thisMark = { label: String(item), value: item };
+        marks.push(thisMark);
+      });
+    } else {
+      marks = predefValues;
+      stepSize = null;
+    }
+  } else {
+    marks = false;
+  }
+  // return components
+  return (
+    <Box>
+      <Typography>{title}</Typography>
+      {labelPosition === "top" ? (
+        <MuiSlider
+          sx={{
+            "& .MuiSlider-markLabel": {
+              position: "absolute",
+              top: "-15px"
+            },
+            marginTop: "20px"
+          }}
+          valueLabelDisplay={displayCurrentValue}
+          step={stepSize}
+          marks={marks}
+          min={min}
+          max={max}
+          onChange={onChange}
+          onChangeCommitted={onChangeCommitted}
+          value={typeof value !== "undefined" ? value : null}
+          orientation={orientation}
+        />
+      ) : (
+        <MuiSlider
+          valueLabelDisplay={displayCurrentValue}
+          step={stepSize}
+          marks={marks}
+          min={min}
+          max={max}
+          onChange={onChange}
+          onChangeCommitted={onChangeCommitted}
+          value={typeof value !== "undefined" ? value : null}
+          orientation={orientation}
+        />
+      )}
+    </Box>
+  );
+}
+
+Slider.propTypes = {
+  /**
+   * If auto the value label will display when the thumb is hovered;
+   * if on, will diplay persistently and if off will never display
+   */
+  displayCurrentValue: PropTypes.oneOf(["auto", "off", "on"]),
```

Comment:
Rename this to match the MUI prop name so users don't have to learn another syntax. MUI prop name is valueLabelDisplay